modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -747,3 +747,20 @@ Ptr<Importer> cv::dnn::createTensorflowImporter(const String&)
 }
 
 #endif //HAVE_PROTOBUF
+
+Net cv::dnn::readNetFromTensorflow(const String &model)
+{
+    Ptr<Importer> tensorflowImporter;
+    try
+    {
+        tensorflowImporter = createTensorflowImporter(model);
+    }
+    catch(...)
+    {
+    }
```

Comment:
@alalek Thank you for your advice. 
I fixed it to returns an empty cv::dnn::Net().